joschahenningsen/TUM-Live
# Enh/983 stream next prev btns

### Motivation and Context
It is unnecessarily hard to navigate to the next stream from the watch page.

### Description
It was requested to add previous and next buttons to the stream page, with optional a streams-overview list.
Both of that is tackled in this PR.

### Steps for Testing

Prerequisites:
- 1 Students

1. Log in
2. Navigate to a Livestream
3. Open the Playlist Sidebar
4. Check if the current stream is marked as playing
5. Check if the other streams are displayed accordingly
6. Check if the Prev and Next button is working

### Screenshots
![Bildschirmfoto 2023-04-19 um 17 51 04](https://user-images.githubusercontent.com/17506411/233131176-f119df16-cefc-4d5f-94e3-36ac03a8dc1d.png)
